### PR TITLE
feat(cli): explain how to configure and retrieve a license

### PR DESCRIPTION
**Pull request by tbouffard**
### Tasks

- [x] Add detailed information about how to retrieve a license. May be added in a page of the application module. This requires to sync with #7 

Closes https://bonitasoft.atlassian.net/browse/BON-367

### CODE REVIEW

File: modules/cli/pages/index.adoc [hunk 1]
```diff
@@ -6,6 +6,16 @@ A Command Line Interface tool (CLI) is used to provision data extracted from Bon
 
 The command line tool is delivered as a package, containing a basic documentation about commands and a sample configuration file.
 
+
+== BPI license
```
Comment (tbouffard) [inline modules/cli/pages/index.adoc:16]: **note:** this is taken from the cli distribution

File: modules/application/images/generate-license.png
Comment (tbouffard) [inline modules/application/images/generate-license.png:1]: **note:** if  you don't like the "square" that tries to highlight the element in the menu, we can use the original screenshot 👇🏿 
We should probably remove the footer. What do you think about this @Yves-Jacq?

Notice that current screenshot is not consistent with the recent screenshots added to other pages of the documentation. They don't display the footer (which includes the Bonitasoft copyright).

generate_license_raw

File: modules/application/pages/license-generation.adoc
Comment (tbouffard) [inline modules/application/pages/license-generation.adoc:1]: **praise:** Previously, all information about the application was in dedicated pages. So I followed this pattern.
After the merge of #11, everything is now in the index page.

@Yves-Jacq should I move the text in the index page as well?

File: modules/application/images/generate-license.png
Comment (Yves-Jacq) [reply modules/application/images/generate-license.png:1]: @tbouffard the footer must be present everywhere

File: modules/application/pages/license-generation.adoc
Comment (Yves-Jacq) [reply modules/application/pages/license-generation.adoc:1]: Yes I thinks it's a better way to navigate

Comment (tbouffard) [reply modules/application/pages/license-generation.adoc:1]: done in 6be9df2

**Comment by tbouffard** [comment]
@Yves-Jacq I have just integrated your feedback.
Can you tell me if this is now OK?

**Review by Yves-Jacq — APPROVED** [review]
OK for me

**Review by Yves-Jacq — APPROVED** [review]
OK for me

**Review by uguy — APPROVED** [review]

**Pull request merged**